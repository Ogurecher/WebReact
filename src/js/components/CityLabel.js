import React, { Component } from "react";

export default class CityLabel extends Component {
    constructor(props) {
        super(props);
    }
    
    render() {
        return (
            <p>
                {this.props.weather.name}
            </p>
        );
    }
}
