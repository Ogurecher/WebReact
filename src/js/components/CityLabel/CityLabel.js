import React, { Component } from 'react';

export default class CityLabel extends Component {

    render() {
        return (
            <p> {this.props.weather.name} </p>
        );
    }
}
