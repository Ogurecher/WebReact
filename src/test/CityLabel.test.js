import React from 'react';
import renderer from 'react-test-renderer';

import CityLabel from '../js/components/CityLabel/CityLabel';

describe('CityLabel', function() {
    test('snapshot renders', function() {
        const weatherStub = {
            name: 'London'
        };

        const component = renderer.create(<CityLabel weather={weatherStub} />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});