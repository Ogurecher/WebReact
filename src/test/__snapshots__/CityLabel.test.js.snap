// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CityLabel snapshot renders 1`] = `
<p>
   
  London
   
</p>
`;
